lnbits/templates/public.html

```diff
@@ -1,7 +1,13 @@
 {% extends "base.html" %} {% block beta %}{% endblock %} {% block drawer_toggle
 %}{% endblock %} {% block drawer %}{% endblock %} {% block toolbar_title %}
-<a href="/" class="inherit">
-  {% if SITE_TITLE != 'LNbits' %} {{ SITE_TITLE }} {% else %}
-  <strong>LN</strong>bits {% endif %}
+<a
+  href="/"
+  class="inherit q-btn q-btn-item non-selectable no-outline q-btn--flat q-btn--rectangle q-btn--actionable q-focusable q-hoverable q-btn--no-uppercase q-btn--wrap q-btn--dense q-btn--active"
+  style="font-size: 20px"
+>
+  {% if USE_CUSTOM_LOGO %}
+  <img height="30px" alt="Logo" src="{{ USE_CUSTOM_LOGO }}" />
+  {%else%} {% if SITE_TITLE != 'LNbits' %} {{ SITE_TITLE }} {% else %}
+  <strong>LN</strong>bits {% endif %} {% endif %}
```

Comment:
Same here: either logo, or page title. Why not both?

Reply:
kinda thought if you have an image logo, you don't get the title!! your question makes sense, not sure it's visually pleasing!